pythran/config.py

```diff
@@ -13,6 +13,11 @@
 logger = logging.getLogger('pythran')
 
 
+def get_include():
+    return os.path.join((os.path.dirname(os.path.dirname(__file__)) or '.'),
+                        'pythran')
```

Comment:
I'm not sure about what this does. I believe the include path should simply be `os.path.dirname(__file__)`? 

All includes in generated code are of the form `#include pythonic/xxx`, so we only need the directory that `pythonic/` is in right?

Reply:
I can't tell fore sure ^^! Looks like under some confiuration, `__file__` has a relative path which makes `direname` returns an empty path, but I can't recall which configuration leads to it. Anyway the above code is a no-op wrt previous behavior so that should be ok.

Reply:
That happens if you run it in IPython or directly as `python config.py`. If it's part of a package, it never fails to give the correct path AFAIK.